
From: Paul Jackson <pj@sgi.com>

It's silly, pointless, and untested to offer CONFIG_CPUSETS for non-SMP
configurations.  Make CPUSETS depend on SMP (even though it doesn't
really).

Signed-off-by: Paul Jackson <pj@sgi.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/init/Kconfig |    1 +
 1 files changed, 1 insertion(+)

diff -puN init/Kconfig~cpusets-config_cpusets-depends-on-smp init/Kconfig
--- 25/init/Kconfig~cpusets-config_cpusets-depends-on-smp	2005-01-10 21:15:15.025169408 -0800
+++ 25-akpm/init/Kconfig	2005-01-10 21:15:15.028168952 -0800
@@ -293,6 +293,7 @@ config EPOLL
 
 config CPUSETS
 	bool "Cpuset support"
+	depends on SMP
 	help
 	  This options will let you create and manage CPUSET's which
 	  allow dynamically partitioning a system into sets of CPUs and
_
